docs/feature_unicode.md

```diff
@@ -80,15 +80,96 @@ void qk_ucis_symbol_fallback (void) { // falls back to manual unicode entry
 Unicode input in QMK works by inputting a sequence of characters to the OS,
 sort of like macro. Unfortunately, each OS has different ideas on how Unicode is input.
 
-This is the current list of Unicode input method in QMK:
+You can set the input method at any time.  You can do this by using a keycode here.  The Input method is listed next to the keycode for reference. 
+
+|Key                       |Aliases  |Input Method  |Description                                        |
+|--------------------------|---------|--------------|---------------------------------------------------|
+|`UNICODE_MODE_OSX`        |`UC_M_OS`|`UC_OSX`      |Sets the input method for MacOS X                  |
+|`UNICODE_MODE_LNX`        |`UC_M_LN`|`UC_LNX`      |Sets the input method for Linux                    |
+|`UNICODE_MODE_WIN`        |`UC_M_WI`|`UC_WIN`      |Sets the input method for Windows                  |
+|`UNICODE_MODE_WINC`       |`UC_M_WC`|`UC_WINC`     |Sets the input method for Windows using WinCompose |
+|`UNICODE_MODE_OSX_RALT`   |`UC_M_OR`|`UC_OSX_RALT` |Sets the input method for MacOS X using RAlt/AltGr |
+
+You can also set the input method via `set_unicode_input_mode(x)`, and this functions the same way as the keycodes above.
 
 * __UC_OSX__: MacOS Unicode Hex Input support. Works only up to 0xFFFF. Disabled by default. To enable: go to System Preferences -> Keyboard -> Input Sources, and enable Unicode Hex.
 * __UC_OSX_RALT__: Same as UC_OSX, but sends the Right Alt key for unicode input
 * __UC_LNX__: Unicode input method under Linux. Works up to 0xFFFFF. Should work almost anywhere on ibus enabled distros. Without ibus, this works under GTK apps, but rarely anywhere else.
 * __UC_WIN__: (not recommended) Windows built-in Unicode input. To enable: create registry key under `HKEY_CURRENT_USER\Control Panel\Input Method\EnableHexNumpad` of type `REG_SZ` called `EnableHexNumpad`, set its value to 1, and reboot. This method is not recommended because of reliability and compatibility issue, use WinCompose method below instead.
 * __UC_WINC__: Windows Unicode input using WinCompose. Requires [WinCompose](https://github.com/samhocevar/wincompose). Works reliably under many (all?) variations of Windows.
 
-At some point, you need to call `set_unicode_input_mode(x)` to set the correct unicode method.  This sets the method that is used to send the unicode, and stores it in EEPROM, so you only need to call this once.
+?> Keep in mind that both methods write to EEPROM, and are loaded each time the keyboard starts. So you only need to hit this once.
+
+### Audio Feedback for Input Mode keycodes
+
+If you have the [Audio feature](feature_audio.md) enabled on the board, you can set "songs" for them to play when pressed, so you have some audio feedback when switching modes.
+
+For instance, you can add these to your `config.h` file.
+
+```c
+#define UNICODE_LINUX_SONG UNICODE_LINUX
```

Comment:
```suggestion
#define UNICODE_SONG_LNX  UNICODE_LINUX
```
I suggest renaming these so that they're consistent with `UNICODE_MODE_LNX` etc.
```c
#define UNICODE_SONG_LNX UNICODE_LINUX
#define UNICODE_SONG_WIN UNICODE_WINDOWS
#define UNICODE_SONG_WINC UNICODE_WINDOWS
#define UNICODE_SONG_OSX COIN_SOUND
#define UNICODE_SONG_OSX_RALT COIN_SOUND
```